lua/illuminate/config.lua

```diff
@@ -37,12 +38,13 @@ end
 
 function M.get()
     return (
-            M.large_file_cutoff() == nil
-            or vim.fn.line('$') <= M.large_file_cutoff()
-            or M.large_file_overrides() == nil
```

Comment:
Why is this being removed?

Reply:
I don't recall my reasoning here.
I can try adding it back.

Reply:
If you add it back I can merge.